examples/v2/project_creation/project.py

```diff
@@ -98,3 +101,38 @@ def GenerateConfig(context):
     })
 
   return {'resources': resources}
+
+def MergeCallingServiceAccountWithOwnerPermissinsIntoBindings(env, properties):
+  """ A helper function that merges the acting service account of the project
+      creator as an owner of the project being created
+  """
+  service_account = ('serviceAccount:{0}@cloudservices.gserviceaccount.com'
+                     .format(env['project_number']))
+  set_creator_sa_as_owner = {
+      'role': 'roles/owner',
+      'members': [
+          service_account,
+      ]
+  }
+  if 'iam-policy' not in properties:
+    return {
+        'bindings': [
+            set_creator_sa_as_owner,
+        ]
+    }
+
+  bindings = []
+  if 'bindings' in properties['iam-policy']:
+    bindings = copy.deepcopy(properties['iam-policy']['bindings'])
+
+  merged = False
+  for binding in bindings:
+    if binding['role'] == 'roles/owner':
+      merged = True
+      if service_account not in binding['members']:
+        binding['members'].append(service_account)
+      break
+
+  if not merged:
+    bindings.append(set_creator_sa_as_owner)
+  return {'bindings': bindings}
```

Comment:
This is ignoring anything else that might be in the policy. Make a copy of properties['iam-policy'], set bindings to the bindings key in there, and then return it. That way any keys other than bindings are also retained (I don't think they exist today, but may in the future).